src/main/java/com/amazonaws/services/sqs/AmazonSQSRequesterClientBuilder.java

```diff
@@ -6,15 +6,18 @@
 import java.util.Optional;
 import java.util.concurrent.TimeUnit;
 
+import static com.amazonaws.services.sqs.AmazonSQSIdleQueueDeletingClient.HEARTBEAT_INTERVAL_SECONDS_DEFAULT;
```

Comment:
nit: I don't think a single reference is worth a static reference, and probably better to be clear at the point of reference where this is coming from

Reply:
Removed the static reference